precompile/contract/utils.go

```diff
@@ -18,6 +18,13 @@ import (
 const (
 	WriteGasCostPerSlot = 20_000
 	ReadGasCostPerSlot  = 5_000
+
+	// Per LOG operation.
+	LogGas uint64 = 375 // from params/protocol_params.go
+	// Gas cost of single topic of the LOG. Should be multiplied by the number of topics.
+	LogTopicGas uint64 = 375 // from params/protocol_params.go
+	// Per byte cost in a LOG operation's data. Should be multiplied by the byte size of the data.
+	LogDataGas uint64 = 8 // from params/protocol_params.go
```

Comment:
I assume this is because of the import loop issue.
I'd like to spend some time later to figure out if there's a better way to handle the config unmarshal, which I believe to be the source of the import loop issue.

Reply:
yup. params package is a really thin ice and can get cyclic with almost everything.